addons/board/static/src/js/dashboard.js

```diff
@@ -145,6 +146,7 @@ var DashBoard = form_common.FormWidget.extend({
             this.action_managers.splice(_.indexOf(this.action_managers, am), 1);
             $container.remove();
             this.do_save_dashboard();
+            data_manager.invalidate();
```

Comment:
I think that this is too early. it should be at the end of the deferred of the do_save_dashboard function